packages/editor/package.json

```diff
@@ -42,10 +42,12 @@
   "types": "src/index",
   "module": "src/index",
   "scripts": {
+    "prepare": "yarn build:generate",
     "clean": "rimraf lib *.tsbuildinfo",
     "build": "tsc -p build.tsconfig.json && yarn build:copyfiles",
     "build:clean": "yarn clean && yarn build",
     "build:copyfiles": "copyfiles -u 1 \"src/**/*.css\" lib",
+    "build:generate": "yarn --cwd ../../packages/protocol",
```

Comment:
This package shouldn't contain anything about the protocol package!

Reply:
gona try to remove it again ... in the intermediate state the builds couldn't work without it. as the this package uses the protocol package. and the inscription.ts sources were not accessible if this build did not run. ... maybe anyway just an intermediate problem, if we're putting the generated code into version control

Reply:
yes but the generation should be triggered in general and not from this package